src/main/resources/META-INF/rewrite/java-version-21.yml

```
@@ -54,6 +54,10 @@ recipeList:
       groupId: net.bytebuddy
       artifactId: byte-buddy
       newVersion: 1.14.11
+  - org.openrewrite.java.dependencies.UpgradeDependencyVersion:
+      groupId: org.modelmapper
+      artifactId: modelmapper
+      newVersion: 3.2.0
```

Typically we use version patterns to not pin ourselves to a particular patch (or even minor) version
```suggestion
      newVersion: 3.2.x
```

---

Sure, updated it